lib/bundler/friendly_errors.rb

```diff
@@ -23,7 +23,7 @@ def log_error(error)
         Bundler.ui.error error.message
       when LoadError
         raise error unless error.message =~ /cannot load such file -- openssl|openssl.so|libcrypto.so/
-        Bundler.ui.error "\nCould not load OpenSSL."
+        Bundler.ui.error "\nCould not load OpenSSL. #{error.class}: #{error}"
```

Comment:
We're showing the error class, message, and backtrace in `--verbose` mode as well, so I figure we can show everything by default and remove the `Bundler.ui.trace error` line below instead?

Reply:
I do not recall ever seeing additional information in verbose mode for this exception therefore I have no objections to removing the verbose functionality.

Reply:
Yeah, I read [your comment in the original issue](https://github.com/bundler/bundler/issues/7192#issuecomment-500029911), but I tried it myself in it does print all the information. Maybe it was a bug in an older version.

We should either recommend rerunning with the `--verbose` flag, or print everything. I tend to prefer the latter since as you point out, without additional information this particular error message is not useful.

Reply:
@p-mongo Are you planning to keep working on this PR, or that I wrap it up myself? Thanks!

Reply:
I have it in my queue but I do not have the bandwidth to tackle it right this moment. Whatever you think is best is fine with me.

Reply:
Ok, kind of the same thing for me :). I guess whoever gets there first can wrap this up. I'll let you know before start working on this so we don't duplicate efforts.

Reply:
@deivid-rodriguez I'm happy to finish this up. I noticed that the `trace` method also displays the error backtrace. Would you like the backtrace to always be printed as well?

Reply:
Yeah, I think in this particular case I would go with "full verbose mode" by default.

Reply:
Done.